### PR TITLE
Feat auth screen

### PR DESCRIPTION
**Pull request by JowelTisso**
This PR is for auth screen and functionality:
* Login and signup screen added
* minor changes in other files
Main file :
* Auth.jsx
* Auth.css
* authHelper.js
* axiosHelper

### CODE REVIEW

**Review by ragini07 — APPROVED** [review]
Liked the modular approach of the file system .

File: src/src/utils/Constants.js [hunk 1]
```diff
@@ -3,3 +3,13 @@ export const LOG_IN = "LOG_IN";
...
+
+// Api
+export const LOG_IN_API = "/api/auth/login";
+export const SIGN_UP_API = "/api/auth/signup";
+export const VIDEOS_API = "/api/videos";
+export const CATEGORIES_API = "/api/categories";
+export const LIKES_API = "/api/user/likes";
+export const WATCHLATER_API = "/api/user/watchlater";
+export const PLAYLIST_API = "/api/user/playlists";
+export const HISTORY_API = "/api/user/history";
```
Comment (manismk) [inline src/src/utils/Constants.js:15]: Suggestion you can put all route links this inside one object of routes and export that object

**Review by manismk — APPROVED** [review]
Good Folder structure and Good UI

File: src/src/utils/Constants.js (hunk 1 above)
Comment (JowelTisso) [reply src/src/utils/Constants.js:15]: Ok will try that

**Pull request merged**